src/pages/learn.tsx

```diff
@@ -0,0 +1,331 @@
+        },
+      ],
+      videoChapters: [
+        {
+          title: "Identity Resolution - Demo",
+          timeStamp: 130,
+        },
+        {
+          title: "Identity Resolution - Responsibilities",
+          timeStamp: 420,
+        },
+        {
+          title: "Identity Resolution - Motivations",
+          timeStamp: 681,
+        },
+        {
+          title: "Panel Discussion",
+          timeStamp: 926,
+        },
+      ],
+    },
+  },
+];
+
+export const LearnPage: FunctionComponent = () => {
+  const postsPerPage = 5;
+  const [currentPage, setCurrentPage] = useState(1);
+  const paginatedPosts = getPaginatedPosts({ posts: webinars, postsPerPage: postsPerPage, currentPage });
+  const totalNoOfPages = getPaginatedPagesTotal({ posts: webinars, postsPerPage: postsPerPage });
+
+  return (
+    <>
+      <Helmet>
+        <meta
+          property="description"
+          content="This series of tech talks is organised by the Infocomm Media Development Authority of Singapore (IMDA) and GovTech Singapore. It comprises six webinars and aims to provide professionals with knowledge on TradeTrust as a digital utility for cross border trade."
+        />
+        <meta
+          property="og:description"
+          content="This series of tech talks is organised by the Infocomm Media Development Authority of Singapore (IMDA) and GovTech Singapore. It comprises six webinars and aims to provide professionals with knowledge on TradeTrust as a digital utility for cross border trade."
+        />
+        <meta property="og:title" content="TradeTrust - An easy way to check and verify your documents" />
+        <meta property="og:url" content={`${window.location.origin}/resources`} />
+        <title>TradeTrust - Resources</title>
+      </Helmet>
+
+      <div className="container py-12">
+        <div className="flex">
+          <div className="w-full text-gray-600">
+            <h1>Learn TradeTrust</h1>
+          </div>
+        </div>
+        <div className="flex flex-wrap py-4">
+          <div className="w-full lg:w-4/12 lg:order-2 lg:pl-3">
+            <div className="lg:sticky lg:top-3">
+              <ResourceLink title="Documentation" resources={documentations} />
+              <ResourceLink title="Source code" resources={sourceCodes} icon="/static/images/Github_Octocat.png" />
+            </div>
+          </div>
+          <div className="w-full lg:w-8/12 lg:order-1 lg:pr-3">
+            {paginatedPosts.map((webinar, index) => (
+              <ResourceWebinar
+                title={webinar.title}
+                description={webinar.description}
+                resource={webinar.resource}
+                key={index}
+              />
+            ))}
+          </div>
+        </div>
+        <div className="w-min mx-auto mb-14">
+          <Pagination totalNoOfPages={totalNoOfPages} currentPage={currentPage} setCurrentPage={setCurrentPage} />
+        </div>
```

Comment:
related, to extract into another `Webinar` component?

Reply:
Done.